sdk/eventhub/eventhubs-checkpointstore-blob/rollup.base.config.js

```diff
@@ -20,9 +20,10 @@ const production = process.env.NODE_ENV === "production";
 
 export function nodeConfig(test = false) {
   const externalNodeBuiltins = ["events", "util", "os"];
+  const additionalExternals = ["keytar"];
```

Comment:
If this should only be excluded in test bundles, shouldn't we check for the `test` parameter to be `true`?

Reply:
True, we could conditionally except it